samples/apps/copilot-chat-app/copilotchat/Storage/ChatMessageRepository.cs

```diff
@@ -30,11 +33,6 @@ public async Task<ChatMessage> FindLastByChatIdAsync(string chatId)
     {
         var chatMessages = await this.FindByChatIdAsync(chatId);
         var first = chatMessages.MaxBy(e => e.Timestamp);
-        if (first is null)
-        {
-            throw new KeyNotFoundException($"No messages found for chat '{chatId}'.");
-        }
-
-        return first;
+        return first is null ? throw new KeyNotFoundException($"No messages found for chat '{chatId}'.") : first;
```

Comment:
I think this can be made even shorter and more legible:

```suggestion
        return first ?? throw new KeyNotFoundException($"No messages found for chat '{chatId}'.") ;
```